Update changelog (#557)
foundry-rs/starknet-foundry: CHANGELOG.md

- Support for custom networks - accounts created on custom networks are saved in `accounts-file` under network's
  chain_id
- `accounts-file` field in Scarb.toml profile

> - Include class hash of the account contract used to create and deploy an account to the `accounts-file`

Reviewer: ```suggestion
- Include the class hash of an account contract in the `accounts-file`
```
wdyt
